lib/taskana-core/src/test/java/acceptance/persistence/UpdateObjectsUseUtcTimeStampsAccTest.java

```diff
@@ -0,0 +1,243 @@
+package acceptance.persistence;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import acceptance.AbstractAccTest;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.TimeZone;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import pro.taskana.Classification;
+import pro.taskana.ClassificationService;
+import pro.taskana.JobService;
+import pro.taskana.Task;
+import pro.taskana.TaskService;
+import pro.taskana.Workbasket;
+import pro.taskana.WorkbasketAccessItem;
+import pro.taskana.WorkbasketService;
+import pro.taskana.WorkbasketType;
+import pro.taskana.exceptions.AttachmentPersistenceException;
+import pro.taskana.exceptions.ClassificationAlreadyExistException;
+import pro.taskana.exceptions.ClassificationNotFoundException;
+import pro.taskana.exceptions.ConcurrencyException;
+import pro.taskana.exceptions.DomainNotFoundException;
+import pro.taskana.exceptions.InvalidArgumentException;
+import pro.taskana.exceptions.InvalidWorkbasketException;
+import pro.taskana.exceptions.NotAuthorizedException;
+import pro.taskana.exceptions.TaskAlreadyExistException;
+import pro.taskana.exceptions.TaskNotFoundException;
+import pro.taskana.exceptions.WorkbasketAlreadyExistException;
+import pro.taskana.exceptions.WorkbasketNotFoundException;
+import pro.taskana.impl.JobServiceImpl;
+import pro.taskana.impl.TaskImpl;
+import pro.taskana.jobs.ScheduledJob;
+import pro.taskana.security.JaasExtension;
+import pro.taskana.security.WithAccessId;
+
+/** Acceptance test for access to timestamps from different timezones. */
+@ExtendWith(JaasExtension.class)
+public class UpdateObjectsUseUtcTimeStampsAccTest extends AbstractAccTest {
```

Comment:
Since the timestamp behaviour should be done globally, do we need specific (complicated) test cases like this?
Can't we just do it with one entity?

Reply:
This testcase was created when we had to annotate each individual timestamp field.
However, I think I doesn't hurt to test the timestamps on all persistent objects...

Reply:
ok